src/components/combo/themes/dark/combo.bootstrap.scss

```diff
@@ -1,12 +1,15 @@
 @use 'sass:map';
 @use 'styles/utilities' as *;
 @use 'components/dropdown/themes/dark/themes' as dropdown;
+@use '../../../input/themes/dark/themes' as input-theme;
```

Comment:
```suggestion
@use 'components/input/themes/dark/themes' as input-theme;
```

Reply:
Why do we need to be declaring the variables for the input here and in all consequent theme files?